stdpopsim/genomes.py

```diff
@@ -50,7 +52,7 @@ def get_chromosome(self, id):
         Returns the chromosome with the specified id.
         """
         for chrom in self.chromosomes:
-            if chrom.id == id:
+            if chrom.id == id or id in chrom.synonyms:
```

Comment:
Should we make this case insensitive?

Reply:
"No" would be my initial reaction (why accept CHR1 as well?), but I could be convinced.

Reply:
`Mt`, `mt`, and even `MT` all seem reasonable to me. And while I don't see an instances here, I'm sure I've seen some UCSC references with captal C for `Chr1` etc., so accepting `chr1` as well would make sense in those cases.

Reply:
yeah i agree with this. we should make this case-insensitive just to add some robustness.

Reply:
In the interest of getting this much merged, moving the discussion to #569